### DISCUSSION TITLE
How to reduce the process for entering a Mozilla Hubs room?

### DISCUSSION THREAD

**Discussion by salahzar** [mozilla/hubs#3018] [General Questions & Discussion] [answered]
Right now when you enter a room you had to follow these steps:

1. ask if you want to enter in stndalone vr or in the room,
2. choose device connected vr or enter on screen,
3. audio setup enter now

and until some days ago you had to do another extra step for the mic being muted or not.
I wonder why you could not set a cookie or make this process a bit more rapid so that you can remember the settings and just ask confirmation only once so that you have all the options selectable in a unique form and still keep security.
This really helps in making the transition from a room to connected rooms much easier.

Sorry if this question has been already asked. I didnt find it, except for teleporting.

**Comment by johnshaughnessy** [comment, marked as answer]
This has been [a long-standing issue](https://github.com/mozilla/hubs/issues/1982) for us, which has been difficult for us to address _in part_ because of how difficult it was for people using desktop and standalone VR devices to enter the room, with the correct audio settings (and across different browsers), successfully. 

While I don't have any guarantees it will be made easier immediately, it _is_ an issue we are very much aware of and will factor into the ongoing work of the [Room UI Redesign](https://github.com/mozilla/hubs/issues/2781).

**Reply by salahzar** [reply]
@johnshaughnessy one solution could be to allow multiple rooms to be put in some of trusted domains so that if you switch from one to another room in the same domain you can maintain the same preferences.
